paper/prep.rb

```diff
@@ -40,6 +41,11 @@
 	keyword_list = metadata['keywords'].join(', ')
 	f << "pdfkeywords = {#{keyword_list}},\n"
 	f << "}\n\n"
+
+	# draft mode
+	if draft
+	  f << "\\usepackage{draftwatermark}\n\n"
```

Comment:
Isn't this _disabling_ the draft?

Reply:
no? Why would that be? I think adding the package adds the watermark

Reply:
I was just curious why the double `\\` at the start of `\\usepackage{draftwatermark}\n\n`? I had interpreted this as the command being commented out... but I now realize that's not how LaTeX works :-)

Reply:
this was escaping special characters like `\n`